src/SFA.DAS.PR.Application.UnitTests/Requests/Queries/GetRequest/GetRequestQueryHandlerTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using SFA.DAS.PR.Application.Mediatr.Responses;
+using SFA.DAS.PR.Application.Requests.Queries.GetRequest;
+using SFA.DAS.PR.Data.Repositories;
+using SFA.DAS.PR.Data.UnitTests.InMemoryDatabases;
+using SFA.DAS.PR.Domain.Entities;
+using SFA.DAS.PR.Domain.Interfaces;
+using SFA.DAS.PR.Domain.Models;
+using SFA.DAS.ProviderRelationships.Types.Models;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.PR.Application.UnitTests.Requests.Queries.GetRequest;
+
+public class GetRequestQueryHandlerTests
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task GetRequestQueryHandler_Handle_ReturnsRequestModel(Request request)
+    {
+        GetRequestQuery query = new GetRequestQuery(request.Id);
+
+        ValidatedResponse<RequestModel?> requestModel;
+
+        Request persistedRequest;
+
+        using (var context = InMemoryProviderRelationshipsDataContext.CreateInMemoryContext(
+            $"{nameof(InMemoryProviderRelationshipsDataContext)}_{nameof(GetRequestQueryHandler_Handle_ReturnsRequestModel)}")
+        )
+        {
+            await context.AddAsync(request, CancellationToken.None);
+            await context.SaveChangesAsync(CancellationToken.None);
+
+            RequestReadRepository requestReadRepository = new(context);
+            GetRequestQueryHandler sut = new(requestReadRepository);
+            requestModel = await sut.Handle(query, CancellationToken.None);
+            persistedRequest = await context.Requests.FirstAsync(CancellationToken.None);
+        }
+
+        requestModel.Result.Should().NotBeNull();
+
+        Assert.Multiple(() =>
```

Comment:
As the models are almost exactly the same apart from RequestId/Id and Name/ProviderName, you could use an 'EquivalentTo' and exclude those two, and then test those as items.  

Also you have a test for ProviderName twice